qemu/tests/cfg/win_virtio_driver_install_by_installer.cfg

```diff
@@ -85,6 +85,12 @@
     Win8, Win2012, Win2012..r2:
         need_reboot = yes
     variants:
+        - installer_version_check:
+            # note that need to have virtio-win rpm pkg installed on host
+            rpm_chk_cmd =  'rpm -q virtio-win'
```

Comment:
I will recommend to define a `package_name = 'virtio-win'` and then reference it everywhere else.